addons/website_slides/models/slide_channel.py

```diff
@@ -623,7 +624,7 @@ def action_view_slides(self):
     def action_view_ratings(self):
         action = self.env["ir.actions.actions"]._for_xml_id("website_slides.rating_rating_action_slide_channel")
         action['name'] = _('Rating of %s') % (self.name)
-        action['domain'] = [('res_id', 'in', self.ids)]
+        action['domain'] = ast.literal_eval(action['domain']) + [('res_id', 'in', self.ids)]
```

Comment:
Would something like this also work?

`action['domain'] = expression.AND([action.get('domain', []), [('res_id', 'in', self.ids)]])`

Seems a bit more elegant, even if the result should be the same.

Reply:
Hey :) It looks more elegant, but  value of  action domain is not tuple nor list, but a string, and this gives AssertionError. If I were to change it, it would look like this: 

action['domain'] = expression.AND([ast.literal_eval(action['domain'])), [('res_id', 'in', self.ids)]]) and this looks even bulkier then previous solution :/

Reply:
Makes total sense, I changed it accordingly :)